crates/core/tedge_agent/src/agent.rs

```diff
@@ -202,6 +202,7 @@ impl Agent {
             &mut mqtt_actor_builder,
             &mqtt_schema,
             self.config.service_type.clone(),
+            self.config.mqtt_topic_root.to_string(),
```

Comment:
The same argument is given twice: first as an `MqttSchema`, second as `String`. Keep only the first one.

Reply:
fixed